pennylane/templates/embeddings/displacement.py

```diff
@@ -28,7 +28,8 @@
 
 @template
 def DisplacementEmbedding(features, wires, method="amplitude", c=0.1):
-    r"""Encodes :math:`N` features into the displacement amplitudes :math:`r` or phases :math:`\phi` of :math:`M` modes,
+    r"""
+    Encodes :math:`N` features into the displacement amplitudes :math:`r` or phases :math:`\phi` of :math:`M` modes,
      where :math:`N\leq M`.
```

Comment:
```suggestion
    r"""Encodes :math:`N` features into the displacement amplitudes :math:`r` or phases :math:`\phi` of :math:`M` modes,
    where :math:`N\leq M`.
```

I think this should fix it. The issue was the single space indentation of 'where', causing Sphinx to treat this as a glossary entry.